sw/device/tests/sim_dv/lc_ctrl_transition_test.c

```diff
@@ -78,9 +80,10 @@ bool test_main(void) {
   CHECK_DIF_OK(dif_lc_ctrl_get_state(&lc, &curr_state));
 
   // The OTP preload image hardcodes the initial LC state transition count to 8.
-  // With each iteration of the test, we increment it.
+  // With each iteration of test, there are two LC_CTRL state transitions.
+  // And the first LC_CTRL state transition is done via external JTAG interface.
   // `kTestIterationCount` starts with 1 in SystemVerilog.
-  const uint8_t LcStateTransitionCount = 8 + kTestIterationCount - 1;
+  const uint8_t LcStateTransitionCount = 8 + 1 + (kTestIterationCount - 1) * 2;
```

Comment:
nit: from style guide: 

s/`LcStateTransitionCount`/`kLcStateTransitionCount`/

Reply:
Thanks Miguel, will update the naming!